### DISCUSSION TITLE
mambaforge3 installation problem

### DISCUSSION THREAD

**Discussion by WilkAndy** [idaholab/moose#20573] [Q&A Getting Started] [answered]
Hey everyone,

I decided to attempt to completely wipe my MOOSE and reinstall everything (due to an ugly non-MOOSE problem i was having).   I just `rm -Rf /opt/moose` and deleted some stuff from my `.zshrc`.  I started to follow the instructions at https://mooseframework.inl.gov/getting_started/installation/conda.html but got stuck on the 2nd step (!!)

```
wil04q@FOURIER-PH: /Users/wil04q/> bash Mambaforge-MacOSX-x86_64.sh -b -p ~/mambaforge3
PREFIX=/Users/wil04q/mambaforge3
Unpacking payload ...
Mambaforge-MacOSX-x86_64.sh: line 362: /Users/wil04q/mambaforge3/conda.exe: Input/output error
Mambaforge-MacOSX-x86_64.sh: line 365: /Users/wil04q/mambaforge3/preconda.tar.bz2: No such file or directory
```

Any ideas how to fix this?

a

**Comment by milljm** [comment, marked as answer]
Can you try using `sh` instead of `bash` (just for funzies). I think I've seen something like this in the past for a bad interpreter parameter.

**Reply by WilkAndy** [reply]
Hi @milljm : I got the same error using `sh` and `bash`.  I'm wondering whether this is a recent error in mambaforge, as there is an issue with identical error message opened 16 days ago at https://github.com/mamba-org/mamba/issues/1543 .    FYI, the command `brew install miniforge` also gives the same error message.

I ended up installing anaconda for python3.7 as per https://docs.anaconda.com/anaconda/install/mac-os/ .  I then followed the instructions at https://mooseframework.inl.gov/getting_started/installation/conda.html , replacing `mamba` with `conda`, viz
```
conda config --add channels https://conda.software.inl.gov/public
conda init
conda create --name moose -q -y
conda activate moose
conda install moose-tools moose-libmesh
conda deactivate
conda activate moose
```
and also adding `conda activate moose` to the bottom of my `.zshrc`.

Do you think that's OK for now?

a

**Reply by GiudGiud** [reply]
@WilkAndy 
you can use conda that should work well too, just a little slower than mamba. 
We'll keep an eye out to see if others are affected with mamba
